src/electron/platform/android/android-scan-results.ts

```diff
@@ -75,7 +75,7 @@ export class AndroidScanResults {
     }
 
     private getAxeResults(): any {
-        return this.rawData?.AxeResults ?? this.rawData;
```

Comment:
V2 moved the AxeResults from rawData to rawData.AxeResults. This line was here to provide automatic handling of either version